lib/tasks/after_tasks.rake

```diff
@@ -874,6 +874,26 @@ namespace :After do
     puts("Added default rating to works: #{updated_works}") && STDOUT.flush
   end
 
+  desc "Reset blurb date (revised_at) on backdated works"
+  task(reset_revised_at_on_backdated_works: :environment) do
+    # Fix all backdated works, as well as works with an arbitrary revised_at
+    # set in the past (possibly because of AO3-6187).
```

Comment:
Do we want to note for future reference that works failing validations (e.g., because they're over the tag limit) will not be fixed? Or should we have a list of work IDs we couldn't fix?

Reply:
Let's do both.